summerofcode/2023.md

```diff
@@ -26,6 +26,46 @@ If you are a project maintainer and consider mentoring during the GSoC 2023 cycl
 
 ## Ideas
 
+### Armada
+
+#### Add Kubectl Plugin for Armada
+
+- Description: Armada is a multi-cluster batch scheduling platform for running high-throughput jobs.  Armada provides a CLI for interacting with jobs, queues and watching events. A common way that Kubernetes users interact with Kubernetes is via kubectl.  This project is to build a kubectl plugin that allows users to submit jobs to Armada using kubectl.
+- Expected outcomes:
+  - Publish a [Krew Plugin](https://krew.sigs.k8s.io/) for Armada
+  - Users can submit jobs, create queues, watch jobs etc using kubectl.
+- Recommend Skills: Go, Kubernetes, Kubectl
+- Mentor(s): Kevin Hannon, @kannon92
+- Expected project size: 100 Hours
+- Difficulty: Easy
+- Upstream Issue (URL): https://github.com/armadaproject/armada/issues/2120
+
+#### Build interfaces around Postgres for Armada
+
+- Description: Open source projects should not be hard coded to a particular Database. Armada currently only allows users to use Postgres.  This project is to build interfaces around our connections to Postgres so we can allow other databases.
+- Expected outcomes:
+  - A interface is created that allows Armada to interact with any SQL database without exposing implementation details of postgres
+  - With an interface, mocking and testing becomes much easier as you can implement mocks for the DB access.  Currently we require a postgres DB for unit testing.
+  - Test coverage could increase
+- Recommend Skills: Go, SQL
+- Mentor(s): Kevin Hannon, @kannon92
```

Comment:
I see you're the mentor for all 3 project ideas.

GSoC program discourages mentoring more than one contributor by a single mentor, given GSoC mentoring would take 3-5 hours of your time weekly.

Also, even when there is a single project, it is encouraged to have a co-mentor (or multiple co-mentors). 

Is there anybody else in the community who can be co-mentors? Ideally, For 3 projects, best setup would be 3 co-mentors sharing all of the mentoring work for all 3 projects.

Reply:
Yea ill find a co-mentor.

Reply:
Added a coworker of mine for one project.